src/index.ts

```diff
@@ -730,6 +733,19 @@ export class Squid {
 
     return response.data.price;
   }
+
+  public async getAllEvmBalances({
```

Comment:
I would rename this function to something like `getAllTokenBalances`, and this must receive 2 addresses, one is evm and the other one is cosmos

Reply:
For Cosmos remember we have multiple address types. So also need to support these.